### PR TITLE
Add Action Cable introduction to release notes

### PR DESCRIPTION
**Pull request by vipulnsward**

### CODE REVIEW

File: guides/source/5_0_release_notes.md [hunk 1]
```diff
@@ -37,9 +37,20 @@ Major Features
 --------------
 
 ### Action Cable
-[Pull Request](https://github.com/rails/rails/pull/22586)
 
-ToDo...
+Action Cable is a new framework in Rails 5. It seamlessly integrates
```
Comment (maclover7) [inline guides/source/5_0_release_notes.md]: new component

**Comment by maclover7** [comment]
This looks pretty good to me, not sure if there's any other big things we want to mention here cc @dhh @javan

File: guides/source/5_0_release_notes.md (hunk 1 above)
Comment (vipulnsward) [inline guides/source/5_0_release_notes.md]: It's a framework. Our overview, etc refer it as framework, see below server-side framework, javascript framework, etc.
I am also basing this on AJ release notes, which was referred to as a framework.

**Comment by dhh** [comment]
Looks good to me.

**Comment by maclover7** [comment]
Backported to `5-0-stable` via d4902bba1e90b78a8e9cbcec0b253e550c98c459, `5-0-0` via d4902bba1e90b78a8e9cbcec0b253e550c98c459 (same commit).

**Pull request merged**